system-contract-dapp-playground/__tests__/hedera/hts-interactions/token-transfer-contract/index.test.ts

```diff
@@ -116,7 +116,8 @@ describe('TokenTransferContract test suite', () => {
         baseContract as unknown as Contract,
         hederaTokenAddress,
         [senderA, senderB],
-        [3, 6, 9]
+        [-18, 3, 6, 9],
```

Comment:
Create a single array constant with an appropriate name for this since it's reused so often